code/lib/preview-api/src/modules/preview-web/UrlStore.ts

```diff
@@ -22,7 +22,7 @@ const getQueryString = ({
   selection?: Selection;
   extraParams?: qs.ParsedQs;
 }) => {
-  const { search = '' } = document.location;
+  const search = (document && document.location && document.location.search) || '';
```

Comment:
I'm ok with this defensive programming, just to be on the safe side.

Minor nit pick: could you use optional chaining here to keep the code concise?

@shilman WDYT?

Reply:
Happy to use optional chaining, I only avoided it to stick with the pattern used here: https://github.com/storybookjs/storybook/blob/next/code/lib/preview-api/src/modules/client-api/queryparams.ts#L7

Reply:
We should start using optional chaining there too 😅

Reply:
@ndelangen do you think it's worth changing this? Or create a separate issue for switching stuff to optional chaining more broadly?